testSrc/com/intellij/plugins/haxe/ide/HaxeCompilerErrorParsingTest.java

```diff
@@ -73,7 +74,9 @@ public void testNMEErrorAbsoluteUnix() {
 
     assertNotNull(compilerError);
     assertEquals(CompilerMessageCategory.ERROR, compilerError.getCategory());
-    assertEquals("/an/absolute/path/HelloWorld.hx", compilerError.getPath());
+    if (Platform.current() == Platform.UNIX) {
+      assertEquals("/an/absolute/path/HelloWorld.hx", compilerError.getPath());
```

Comment:
Hmm,  I wonder if Platform.current() == Platform.UNIX is true for OS X?  I should be able to test that tomorrow.